assets/js/components/data/index.js

```diff
@@ -255,10 +226,48 @@ const dataAPI = {
 			// Resolve any returned data requests, then re-request the remainder after a pause.
 		} ).catch( ( err ) => {
 			// Handle the error and give up trying.
-			console.log( 'error', err ); // eslint-disable-line no-console
+			console.warn( 'Error caught during combinedGet', err ); // eslint-disable-line no-console
 		} );
 	},
 
+	handleWpError( error ) {
+		const { code, data } = error;
+
+		if ( ! code || ! data ) {
+			return;
+		}
```

Comment:
I wonder whether this check is a bit weak. An actual API response could potentially include `code` and `data` parameters as well. Maybe we should at least check for `message` too to make it a bit more likely, or we could include something else to make it truly reliable?

Reply:
I've updated this to be a bit more robust so that we ensure the response has all 3 keys that all WP_Errors have. Unfortunately, we can't be 100% sure (even if we did update the method in REST_Routes although I agree this would be the best) because this same method is used for non-batch responses as well, which won't get any extra handling, so we need to do the best we can with WP_Error responses from core.